web/pgadmin/browser/server_groups/servers/pg_timetable/__init__.py

```
@@ -0,0 +1,628 @@
+                'ignore_error': 'ignore_error',
+            }
+            if 'kind' in task:
+                sets.append("kind = %s::timetable.command_kind")
+                params.append(task['kind'])
+            for frontend_field, db_field in field_map.items():
+                if frontend_field in task:
+                    sets.append(f"{db_field} = %s")
+                    params.append(task[frontend_field])
+            if sets:
+                params.extend([tid, chain_id])
+                sql = f"UPDATE timetable.task SET {', '.join(sets)} WHERE task_id = %s AND chain_id = %s"
```

Ruff (0.15.21)

[error] Possible SQL injection vector through string-based query construction

(S608)